apple/Sources/UniFFI/ferrostar.swift

```diff
@@ -3092,6 +3092,7 @@ public func FfiConverterTypeRouteDeviationTracking_lower(_ value: RouteDeviation
 
 public enum RouteRequest {
     case httpPost(url: String, headers: [String: String], body: Data)
+    case httpGet(url: String, headers: [String: String])
```

Comment:
Correct; this is what lets us do it in a consistent manner across platforms. This isn't quite as important when you're bringing your own code (in a `CustomRouteProvider`, which can do literally anything), and I guess it does make things slightly less ergonomic when implementing your own HTTP-based route provider _in platform code_. I think the tradeoff is worth it though, since it means we can add any number of reasonably common providers (ex: GraphHopper) directly in the core. In those cases, we'll expose convenience initializers like we do for Valhalla already and we won't have to write logic to construct HTTP requests and execute them in a bespoke manner for each platform.